processing/src/main/java/io/druid/segment/virtual/SingleStringInputDimensionSelector.java

```diff
@@ -139,7 +139,7 @@ public String lookupName(final int id)
   @Override
   public boolean nameLookupPossibleInAdvance()
   {
-    return true;
+    return selector.nameLookupPossibleInAdvance();
```

Comment:
This is not a bug, since the constructor verifies that `selector.nameLookupPossibleInAdvance()` returns true. (Otherwise, the optimization this class is trying to do is pointless.)

Reply:
Thanks, fixed